nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVRecordSetWriter.java

```diff
@@ -48,6 +54,7 @@
     @Override
     protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
         final List<PropertyDescriptor> properties = new ArrayList<>(super.getSupportedPropertyDescriptors());
+        properties.add(CSVUtils.EXPLICIT_COLUMNS);
```

Comment:
Alright, I modified the capability descriptors for CSVReader and CSVRecordSetWriter. Take a look and let me know what you think =)